ISISSoftwareServices/AutoreductionMonitor
# Containerisation of System Monitoring Database

### Summary of work
Creation of Dockerfile, database schema image and documentation detailing containerisation of system monitoring database.

### Where
`database_containerisation/`

### How to test your work
* Ensure documented instructions are clear and concise.
* Manual Testing on local machine and/or SCD Cloud 

    For  manual testing on SCD Cloud, you will need to first follow the instructions detailed in [repository Wiki](https://github.com/ISISSoftwareServices/AutoreductionMonitor/wiki/Containerisation_setup) - Issue #5 
    * Ensure that the container created following README.md instructions, correctly mounts a data volume and contains tables from schema file.
    * Ensure that after adding new data, if you kill and remove the running database container, on re-creating the 
container again, data last added is persisted.

### Additional comments
* Please note that you will need Docker installed to test this PR
* As this is only for local use right now, credentials are visible in DockerFile. This will be templated using a credentials.ini file at a later point. 
* Travis will state the build has failed - This is simply because no tests have been added yet as the repository is very new.

Fixes #4 #5
